cli/commands.go

```diff
@@ -305,18 +305,17 @@ func PrintProvides(device *dev.DeviceManager) error {
 	return nil
 }
 
+//The channel gets initialized at boot
+var SignalHandlerChan chan os.Signal
```

Comment:
I understand that SignalHandlerChan here in commands needs to Capitalized "C" to be exported. But should I follow the standard camel case on the local SignalHandlerChan varables, like on line 142 in cli_test.go and 29 in main.go. Or should I have all capitalized "C" to be consistent?